specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.ApiManagement/ApiManagement.tsp

```diff
@@ -130,6 +130,70 @@ model ApiManagementGatewayApiRemovedEventData {
   ...ApiGatewayApiBase;
 }
 
+/** Schema of the Data property of an EventGridEvent for a Microsoft.ApiManagement.CircuitBreaker.Opened event. */
+model ApiManagementCircuitBreakerOpenedEventData {
+  /** Name of the backend for which the circuit has opened. */
+  backendName: string;
```

Comment:
Is this a host name?

Reply:
No, this is a customer specified name for their backend